libr/core/cconfig.c

```diff
@@ -3151,6 +3151,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETCB ("cfg.sandbox", "false", &cb_cfgsanbox, "Sandbox mode disables systems and open on upper directories");
 	SETPREF ("cfg.wseek", "false", "Seek after write");
 	SETCB ("cfg.bigendian", "false", &cb_bigendian, "Use little (false) or big (true) endianness");
+	SETI ("cfg.cpuid", -1, "Run on cpuid");
```

Comment:
this can be confusing with the cpuid instruction, better use something like `cfg.affinity` or `cfg.cpuaffinity`